readthedocs/notifications/constants.py

```diff
@@ -10,25 +8,20 @@
 REQUIREMENT = 2
 ERROR = 3
 
-LEVEL_MAPPING = {
-    INFO: message_constants.INFO_PERSISTENT,
-    WARNING: message_constants.WARNING_PERSISTENT,
-    REQUIREMENT: message_constants.WARNING_PERSISTENT,
-    ERROR: message_constants.ERROR_PERSISTENT,
-}
+# TODO: use Enum here for these ones
+# Status
+UNREAD = "unread"
+READ = "read"
+DISMISSED = "dissmissed"
+CANCELLED = "cancelled"
 
-# Message levels to save the message into the database and mark as read
-# immediately after retrieved (one-time shown message)
-DEBUG_NON_PERSISTENT = 100
-INFO_NON_PERSISTENT = 101
-SUCCESS_NON_PERSISTENT = 102
-WARNING_NON_PERSISTENT = 103
-ERROR_NON_PERSISTENT = 104
+# Type
+ERROR = "error"
+WARNING = "warning"
+INFO = "info"
+NOTE = "note"
+TIP = "tip"
 
-NON_PERSISTENT_MESSAGE_LEVELS = (
-    DEBUG_NON_PERSISTENT,
-    INFO_NON_PERSISTENT,
-    SUCCESS_NON_PERSISTENT,
-    WARNING_NON_PERSISTENT,
-    ERROR_NON_PERSISTENT,
-)
+# Icons
+SOLID = "solid"
+DUOTONE = "duotone"
```

Comment:
I noticed this elsewhere, but I think we should probably just change the notification icon to a icon_classes. There are other classes used too, `fa-swap-opacity`/etc. In this case, there is no separate field for icon type, just:

```Notification.icon_classes = "fas fa-user-group fa-swap-opacity"```